### PR TITLE
Garnett A ton of padding

### PR DESCRIPTION
**Pull request by superfrank**
Garnett meta padding changes and allowing removal of byline image div when it's not displayed in article.

paddington

### CODE REVIEW

File: common/app/views/fragments/contentMeta.scala.html [hunk 1]
```diff
@@ -22,9 +22,8 @@
 @metaBody() = {
 
     @if(item.content.showCircularBylinePicAtSide && !item.content.isImmersive) {
-        <div class="media__img meta__image">
+
```
Comment (regiskuckaertz) [inline common/app/views/fragments/contentMeta.scala.html]: At this point, are we comfortable making a change that will impact the non-Garnett visitors too?

Comment (regiskuckaertz) [reply common/app/views/fragments/contentMeta.scala.html]: (and I saw you added it in the template, which makes much more sense imho – no phantom tag!)

**Review by regiskuckaertz — APPROVED** [review]
Yeah!

File: common/app/views/fragments/contentMeta.scala.html (hunk 1 above)
Comment (superfrank) [reply common/app/views/fragments/contentMeta.scala.html]: So, this stops an empty image div from appearing on articles when they don't have one. It's a fix that that should be good for both.

Comment (superfrank) [reply common/app/views/fragments/contentMeta.scala.html]: exactly!

**Pull request merged**